packages/carbon-web-components/.storybook/_container.scss

```
@@ -7,7 +7,22 @@
 // LICENSE file in the root directory of this source tree.
 //
 
-@use '@carbon/styles' as *;
+@use '@carbon/styles/scss/config' as *;
+@use '@carbon/styles/scss/utilities/convert' as *;
+@use '@carbon/styles/scss/utilities';
+@use '@carbon/styles/scss/fonts';
+@use '@carbon/styles/scss/theme' as *;
+@use '@carbon/styles/scss/themes' as *;
+
+@import '@carbon/type/scss/type';
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@import '@carbon/type/scss/font-face/serif';
+
+@include carbon--type-reset();
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+@include carbon--font-face-serif();
```

```suggestion
@use '@carbon/styles/scss/config' as *;
@use '@carbon/styles/scss/utilities/convert' as *;
@use '@carbon/styles/scss/utilities';
@use '@carbon/styles/scss/fonts';
@use '@carbon/styles/scss/theme' as *;
@use '@carbon/styles/scss/themes' as *;
@use '@carbon/styles/scss/type' as *;

@include reset();
```

If importing all the styles is conflicting with component-specific styles, then we should just import type styles from `@carbon/styles`. All the `import`/`include` are from v10. The v10 styles were also interfering with the v11 reset styles, that's why I made that initial change.

---

that works! thanks @kennylam